plugins/modules/mysql_replication.py

```diff
@@ -131,6 +131,12 @@
       L(MySQL encrypted replication documentation,https://dev.mysql.com/doc/refman/8.0/en/replication-solutions-encrypted-connections.html).
     type: str
     aliases: [master_ssl_cipher]
+  primary_ssl_verify_server_cert:
+    description:
+    - Same as mysql variable.
+    type: bool
+    default: false
+    version_added: '3.4.5'
```

Comment:
```suggestion
    version_added: '3.4.1'
```

Reply:
sorry it should be `3.5.0`, it was a typo. We follow semver